ci/release/Jenkinsfile

```diff
@@ -34,10 +33,17 @@ if (currentBuild.getBuildCauses().toString().contains('BranchIndexingCause')) {
 }
 
 def manualRelease = currentBuild.getBuildCauses().toString().contains( 'UserIdCause' )
+def cronRelease = currentBuild.getBuildCauses().toString().contains( 'TimerTriggerCause' )
 
 // Only do automatic release on branches where we opted in
-if ( !manualRelease && !RELEASE_ON_PUSH ) {
-	print "INFO: Build skipped because automated releases are disabled on this branch. See constant RELEASE_ON_PUSH in ci/release/Jenkinsfile"
+if ( !manualRelease && !cronRelease ) {
+	print "INFO: Build skipped because automated releases on push are disabled on this branch."
+	currentBuild.result = 'NOT_BUILT'
+	return
+}
```

Comment:
I think we should also disable the "build on push" option, to avoid the messy build history with tons of "not built" runs.

Maybe like this? https://stackoverflow.com/a/72917011/6692043

Makes sense to keep this check just in case, though.